### PR TITLE
Add services.yaml

### PR DESCRIPTION
**Pull request by chueatwork**
First draft services.yaml

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- Athena
+- Batch
+- Comprehend Medical
```
Comment (jflasher) [inline services.yaml:3]: @chueatwork think we can drop this to just Comprehend? It'll be a very long tag otherwise.

**Comment by jflasher** [comment]
Assuming we're good with an initial services list, we should added the Services block to a few Tutorial entries to get this rolling. It'd look like below. @chueatwork do you want to try this for some of the ones you've been looking at?

```YAML
  Tutorials:
    - Title: How to scale sentiment analysis using Amazon Comprehend, AWS Glue and Amazon Athena
      URL: https://aws.amazon.com/blogs/machine-learning/how-to-scale-sentiment-analysis-using-amazon-comprehend-aws-glue-and-amazon-athena/
      AuthorName: Roy Hasson
      AuthorURL: https://twitter.com/royhasson
      Services:
        - Athena
        - SageMaker 
```

File: services.yaml (hunk 1 above)
Comment (chueatwork) [reply services.yaml:3]: Yeah I was going to ask about character limit.

Did I really leave of SNS, wth

**Comment by chueatwork** [comment]
Yup!

File: services.yaml (hunk 1 above)
Comment (jflasher) [reply services.yaml]: No hard limit, but just figured it'd be really long. Thanks!

**Comment by jflasher** [comment]
@chueatwork can you also update the README to call out the Services field and add example to the example entry there?

File: README.md [hunk 2]
```diff
@@ -67,11 +68,13 @@ The metadata required for each dataset entry is as follows:
...
 |**DataAtWork [> Tutorials, Tools & Applications, Publications] > Title**|String|The title of the tutorial, tool, application, or publication that uses the data.|
 |**DataAtWork [> Tutorials, Tools & Applications, Publications] > URL**|URL|A link to the tutorial, tool, application, or publication that uses the data.|
 |**DataAtWork [> Tutorials, Tools & Applications, Publications] > AuthorName**|String|Name of person or entity that created  the tutorial, tool, application, or publication.|
 |**DataAtWork [> Tutorials, Tools & Applications, Publications] > AuthorURL**|String|(Optional) URL for person or entity that created the tutorial, tool, application, or publication.|
+|**DataAtWork [> Tutorials] > Services** (Optional)|String|For tutorials only. List AWS Services applied in your tutorial. A list of supported AWS services is maintained in the [services.yaml](services.yaml) file in this repo. If you want to recommend a resource that is not included in [services.yaml](services.yaml), please submit a pull request to add it to that file.|
```
Comment (jflasher) [inline README.md:77]: Can you add `Services` examples to the sample entry at https://github.com/awslabs/open-data-registry/blob/17ab6466a63433318714b6e3f458d743fcf256a0/README.md#example-entry

Comment (chueatwork) [reply README.md:77]: Added Services: EC2 to fNEXRAD on EC2 tutorial

**Pull request merged**